test/unit_tests/braket/aws/test_aws_device.py

```diff
@@ -305,9 +319,9 @@ def test_aws_session_in_another_qpu_region(
     aws_session.get_device.return_value = MOCK_GATE_MODEL_QPU_1
 
     creds = Mock()
-    creds.access_key = "access key"
-    creds.secret_key = "secret key"
-    creds.token = "token"
+    creds.access_key = "foo"
```

Comment:
Wondering what's up with this change?

Reply:
I added creds to the fixture too, and just wanted to distinguish the ones here. Even though the fixture creds aren't used in this test, in case it gets modified down the line, I don't want a pass simply because the creds are coicidentally the same.

Reply:
Believe it or not, AWS Style advises not to use "foo" and those other tokens, due to their unsavory origins in military profanity.  I realize they are almost universal in the industry. If you can possibly think of others, that would be better, because customers will see this example, correct?